plugins/google_search/help.md

```diff
@@ -75,6 +115,7 @@ This plugin does not contain any troubleshooting information.
 
 # Version History
 
+* 2.0.0 - Bug fix for Search action | Removal of retired Search action parameter | Update to Insight Connect SDK
```

Comment:
```suggestion
* 2.0.0 - Bug fix and removal of "start" input parameter in the Search action | Update t Insight Connect SDK to version 4
```

Reply:
Have updated this now with reference to the "only_standard" parameter instead.